### PR TITLE
Add ci script && Dockerfile for playground

### PR DESCRIPTION
**Pull request by DerTiedemann**

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+          password: ${{ secrets.docker_pass }}
+      - name: Set tag variables
+        id: setTagVariables
+        run: |
+          BRANCH_NAME=${{ github.ref_name }}
+          if [[ "${{ github.ref_name }}" == "main" ]]; then
+              echo "::set-output name=stable::c4tplatform/camino-docs:stable"
+          fi
+          if [[ "${{ github.ref_name }}" == "dev" ]]; then
+              echo "::set-output name=stable::c4tplatform/camino-docs:dev"
```
Comment (Noctunus) [inline .github/workflows/build.yaml:66]: I'm not familiar with the syntax here, but is "name=stable" right for the dev deploy?

**Review by Noctunus — APPROVED** [review]
Dockerfile looks good, the build script has some open questions - if they can be resolved and everythings intended then it should be fine.

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        name: Login to DockerHub
+        with:
+          username: ${{ secrets.docker_username }}
+          password: ${{ secrets.docker_pass }}
+      - name: Set tag variables
+        id: setTagVariables
+        run: |
+          BRANCH_NAME=${{ github.ref_name }}
+          if [[ "${{ github.ref_name }}" == "main" ]]; then
+              echo "::set-output name=stable::c4tplatform/camino-docs:stable"
```
[the remark this answers is not in the document]
Comment (DerTiedemann) [reply .github/workflows/build.yaml:63]: One is for the static tag {stable, dev}, and the other one is to specifically identify a previous build, it makes it easier to rollback in emergency situations

File: .github/workflows/build.yaml (hunk 1 above)
Comment (DerTiedemann) [reply .github/workflows/build.yaml:66]: yes, as it just a variable name that is used in the build and push action

**Pull request merged**